apps/decoder_json/lib/decoder/json.ex

```diff
@@ -0,0 +1,31 @@
+defmodule Decoder.Json do
+  use Definition, schema: Decoder.Json.V1
+
+  @derive Jason.Encoder
+  defstruct chunk_size: 1000
+
+  defimpl Decoder do
+    def lines_or_bytes(_t), do: :line
```

Comment:
Why do both json and json_lines return `:line` ?

Reply:
That function has a different meaning, we probably should rename it, the purpose of that function is to tell the extractors if the decoder expects lines of text of raw bytes.   Which reminds me I don't think I am calling this function yet.